# Live-Organismal Transcriptomics
Live-Organismal Transcriptomics is a method for profiling the real time-series of gene expression within individual living cells in single organisms. To use these notebooks, you willã€€some packages and tools accessible through both pip or conda.

## Main analysis with Live-organism transcriptome data
The analysis presented in the notebook, AnalysisofLiveOrganismalData.ipynb, is for Figs. 3B-G, 4B-E, 4G and Supplementary Figs. 9C-D, 10B-G, 11A-D.

## Comparison of principal variance and Gini index
The analysis presented in the notebook, PrincipalVariance_vs_GiniIndex.ipynb, is for Fig. 4F.

## Image analysis with Fucci data
The analysis presented in the notebook, IdentificationTracing_FucciCell.ipynb, is for Figs. 1A, 2D and Supplementary Figs. 2A-E, 3A-D, 5A, 5F.

## Find variavble genes
The analysis presented in the notebook, FindVariableGenes.ipynb, is for Supplementary Fig. 4G.

## scGRN and GRN prediction and estimation of transcription and degradation rates by dynGENIE3
The analysis in a notebook, GRNestimationbyDynGENIE3.ipynb, is for Figs. 1C and 1H and Supplementary Figs. 2A-E, 3A-D, 5B-F, 8.
dynGENIE3_DegradationRate.py and dynGENIE3_TimeDifference.py are used for the prediction of GRNs based on degradation rate and real velocity, respectively.

## Batch collection by Seurat
BatchColl.R was used for batch correction of scRNA-seq data and live-cell transcriptome data. The corresponding diagram is Supplementary Figs. 4C-F.

## Correlation matrix of scGRNs and distance matrix of Fucci trajectories
The analysis presented in the notebook, CorrMatrixofscGRNs_DistMatrixFucci.ipynb, is for Figs. 1D-E.

## Regression with scGRNs and Fucci trajectories
The analysis presented in the notebook, RegressionWithscGRNandFucci.ipynb, is for Figs. 1H, 2A-C and Supplementary Fig. 5D.

## Estimation of RNA velicity by velocyto
The analysis presented in the notebook, tSNEembedding_Velocyto.ipynb, is for Fig. 1H and Supplementary Fig. 7.

